src/search/DownwardFiles.cmake

```diff
@@ -761,6 +761,7 @@ fast_downward_plugin(
         pdbs/pattern_collection_generator_multiple_random
         pdbs/pattern_collection_generator_multiple
         pdbs/pattern_collection_generator_systematic
+        pdbs/pattern_database_factory
```

Comment:
I don't remember what we discussed about this. If we decided against the rename, can you comment what the reason against this was and click "resolve conversation", so it doesn't show as an open discussion?

Reply:
I think you wanted it to be renamed. I don't have a strong opinion so I'll do it.

Reply:
My suggestion would be to rename it if it should take the role of a builder in the long term. Otherwise, we'll have to rename it later (either from builder to factory or the other way). If this is supposed to stay a private class, I think sticking with factory is fine.